apps/camera/style/recording-timer.css

```diff
@@ -88,14 +98,3 @@
   right: 0;
   left: auto;
 }
-
-/**
- * deg90,
- * deg180
- */
-
-.deg90 .recording-timer:before,
-.deg180 .recording-timer:before {
-  right: 2rem;
-  left: auto;
-}
```

Comment:
Syle here is causing red icon and recording time overlaps when phone is rotated 90 and 180 deg